src/main/kotlin/carracing/main.kt

```diff
@@ -0,0 +1,38 @@
+package carracing
+
+const val VIEW_STRING_CAR = "-"
+const val VIEW_STRING_SEPARATOR = "\n"
+const val FORWARD_INT = 4
+const val MIN_RANDOM_INT = 1
+const val MAX_RANDOM_INT = 9
+
+fun main() {
+    println("자동차 대수는 몇 대인가요?")
+    val carCount: String = readln()
+    println("시도할 횟수는 몇 회인가요?")
+    val movementTryCount: String = readln()
+    val inputDto = InputDto(carCount, movementTryCount)
+
+    val randomIntCarRacingManager = RandomIntCarRacingManager.fromInputDto(
```

Comment:
필드명에  random이 붙을 필요는 없을 것 같아요!

```suggestion
    val carRacingManager = RandomIntCarRacingManager.fromInputDto(
```